tests/gengedcomstrong/test1.llscr

```diff
@@ -2,7 +2,7 @@ y
 urTESTNAME.ged
 y
 y
-rtest1.ll
+rtest1
```

Comment:
Shouldn't this be rTESTNAME.ll ?

Reply:
for .ll files either way is supported. F flip-flop back and forth on this issue . Do we have one rule, or do we document the behavior.   .ll files are the exception to the rule because LLPROGRAMS is set to load them. I changed CreateTests to say both ways are ok for .ll files.